lookups/lookup-get-basic-example-1/lookup-get-basic-example-1.3.x.js

```diff
@@ -0,0 +1,14 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
```

Comment:
This one could use the promise api. Also I tried to run it and I got this:

http://d.pr/i/vq6z